packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -403,6 +405,11 @@ rocksdb::Status ArbCore::reorgToMessageCountOrBefore(
             tx.checkpointDelete(checkpoint_it->key());
 
             checkpoint_it->Prev();
+            if (!checkpoint_it->status().ok()) {
```

Comment:
If checkpoint_it->IsValid() is true, that means there's no error, so I think this can be taken out of the loop and put before the IsValid check after the loop

Reply:
I was looking at old documentation, isn't actually needed, so removed the code